src/components/UsersList/UsersList.tsx

```diff
@@ -50,37 +70,57 @@ export function UsersList({
     }
   }
 
-  function removeUserCallback(user: UserObj) {
-    setRemovedUserGuids([...removedUserGuids, user.guid]);
-    openModal(user);
+  // Sorting
+  interface SortParamsTable {
+    [colName: string]: string;
   }
+  const sortParamsTable = {
+    'account name': 'username',
+    'organization roles': 'orgRolesCount',
+    'access permissions': 'spaceRolesCount',
+    expires: 'daysToExpiration',
+    'last login': 'lastLogonTime',
+  } as SortParamsTable;
 
-  const [removedUsername, setRemovedUsername] = useState('');
+  function usersSorted(usersList: Array<UserOrgPage>): Array<UserOrgPage> {
```

Comment:
[question] I am missing where `usersSorted` is being used in this file. I was initially curious about the difference in syntax between that and `onSortAction` but now I wonder if it's actually a leftover?

Reply:
It's being used on [Line 116](https://github.com/cloud-gov/cg-ui/blob/ab56c92c80b9e73a808a71e8bc92744104e8eb17/src/components/UsersList/UsersList.tsx#L116) where the sorting + filtering happens to `currentUsers`

Reply:
`onSortAction` handles setting the sorting-related state configs, like `asc` or `desc`. If the wording can be clearer to distinguish the two, lmk!